app/models/task.rb

```diff
@@ -7,6 +7,10 @@ class Task < ApplicationRecord
 
   belongs_to :project
 
+  def form_id
+    "#{project.id}-#{id}"
+  end
```

Comment:
is it really really needed in the model? maybe a view helper would be a better place?